python/paddle/v2/fluid/io.py

```diff
@@ -241,6 +269,9 @@ def save_inference_model(dirname,
             "fetch_var_names": fetch_var_names
         }, f, -1)
 
+    prepend_feed_ops(inference_program, feeded_var_names)
```

Comment:
We can remove `Line 265 - 270` now, and change the implementation of `load_inference_model`.

Reply:
Will do this in the next PR. Thanks!